cypress/support/parseHTML.ts

```diff
@@ -0,0 +1,11 @@
+import { htmlWriter } from "./HtmlWriter";
+
+const corePath = 'http://localhost:3030/libdev/plot-data.js';
```

Comment:
Yes it is since it is created in package.json with http server package with this specific local host
`"test": "npm run dev & start-server-and-test start http-get://localhost:3030 cy:run-actual",
    "base-test": "npm run dev & start-server-and-test start http-get://localhost:3030 cy:run-base",
    "cyopen": "npm run dev & start-server-and-test start http-get://localhost:3030 cy:run-open"`

Reply:
That's what I thought when I saw these lines later on!